index.html

```diff
@@ -305,13 +305,33 @@ <h2>Languages</h2>
                         </div>
 
                     </div>
+            </section>
 
+            <!--Added by Sandeep (mailsg0912@gmail.com)-->
 
-                   </section>
-               <section  id="contact"></section>
-
-            </div>
-
+            <section id="contact">
+                <div class="contact-shape">
+                </div>
+                <div class="contact-form">
+                    <h2>Contact me</h2>
+                    <div class="contact-description">
+                        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                            I’d love to help with it!
+                        </p>
+
+                        <form action="#" method="">
+                            <input type="text" name="name" placeholder=" alexandra" maxlength="30" required>
+                            <input type="email" name="email" placeholder=" alexandra@gmail.com" required>
+                            <textarea name="message" maxlength="50" required placeholder="Write your message here"></textarea>
```

Comment:
- The max length for the textarea should be 500 characters. kindly update this.

Reply:
### Design

- Please, kindly improve your design to improve the user experience of your website. This is how it looks currently on mobile:

![Screenshot from 2023-04-04 09-36-54](https://user-images.githubusercontent.com/59397159/229736485-aaed6ccb-1786-403a-8f23-e107f4625f76.png)

Reply:
Fixed.